content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,362 @@
+==> Status: finished
+==> Started: 2021-11-26T12:46:17
+==> Finished: 2021-11-26T12:46:23
+==> Logs:
+==> Step: fitdata
+==> Workflow ID: 4455a6b2-3d94-4694-ae99-e493327cd53f
+==> Compute backend: Kubernetes
+==> Job ID: reana-run-job-2da81be7-5108-41cd-8b7c-ffc0f081b3bd
+==> Docker image: reanahub/reana-env-root6:6.18.04
+==> Command: root -b -q 'code/fitdata.C("results/data.root","results/plot.png")'
+==> Status: finished
+==> Started: 2021-11-26T12:46:23
+==> Finished: 2021-11-26T12:46:33
+==> Logs:
+```
+
+### Removal of support for Python 2.7
+
+Following the discontinuation of Python 2.7 official support, REANA
+0.8 release series stopped supporting Python 2.7 version. Please
+upgrade your `reana-client` installation to use at least Python 3.6.
+
+## What's new for the administrators?
+
+### CPU and Disk quota accounting
+
+REANA 0.8 release series allows cluster administrators to set certain
+CPU and Disk quota usage limits for individual users. This can be set
+by Helm configuration option `quota.enabled`.
+
+Please note that if you do enable CPU and Disk quota monitoring, you
+may want to further configure `quota.termination_update_policy` to
+decide whether the CPU and Disk quota usage consumption for users will
+be calculated immediately after workflow terminates or via a periodic
+cron jobs.
+
+### Improved cluster performance
+
+The cluster performance when running numerous concurrent workflows was
+considerably improved. We have tested running up to thousand of
+concurrent workflows running on clusters of up to three hundred
+nodes. There are new Helm configuration options that allow to
+fine-tune REANA cluster components in these conditions, such as:
+
+- ``reana_server.environment.REANA_WORKFLOW_SCHEDULING_POLICY`` allows
+  to set workflow scheduling policy. The value "fifo", meaning
+  "first-in first-out", will execute worfklows as they are coming. The
+  value "balanced" will look at how many workflows a particular user
+  launches. If user A uses system heavily, and user B submits new
+  workflows, the system will prefer to schedule the workflow from user
+  B before continuing with workflows of the user A. The "balanced"
+  scheduling policy will also look at the workflow DAG complexity. If
+  a new worklow C would like to launch 10 jobs at the start, and the
+  workflow D only 2 jobs, and the cluster is busy and the current
+  capacity cannot accept more than 4 jobs at the given time, the
+  workflow D would be preferred over workflow C.
+
+- ``reana_server.environment.REANA_RATELIMIT_GUEST_USER`` and
+  ``reana_server.environment.REANA_RATELIMIT_AUTHENTICATED_USER``
+  allow to set REST API rate limit values.  If your cluster is very
+  busy, you can use values such as "2000 per second" enabling heavy
+  traffic from running a high amount of concurrent workflows.
+
+- ``reana_server.environment.REANA_SCHEDULER_REQUEUE_SLEEP`` allows to
+  set sleep time between processing queued workflows. The default
```

Comment:
allows to set -> "allows setting" or "allows us/you to set"

Reply:
Missing.

Reply:
Added.